python/cuda_cccl/cuda/cccl/parallel/experimental/algorithms/_histogram.py

```diff
@@ -184,6 +184,20 @@ def histogram_even(
     num_samples: int,
     stream=None,
 ):
+    """
+    Performs device-wide histogram computation with evenly-spaced bins using the single-phase API.
```

Comment:
```suggestion
    Performs device-wide histogram computation with evenly-spaced bins.
```

No need to make any references to "single-phase API" in the docs. Ditto below.